08.ls_object/format.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Ls
+  class Format
+    DISPLAY_COLUMN_NUMBER = 3
+
+    def initialize(dir_files)
+      @dir_files = dir_files
+    end
+
+    def shortformat
+      sliced_dir_files = slice_dir(@dir_files)
+
+      transposed_sliced_dir_files = sliced_dir_files.transpose
+
+      transposed_sliced_dir_files.each do |array|
+        display_width = @dir_files.max_by(&:length).length + 10
+
+        array.each do |display|
+          print display + ' ' * (display_width - display.length)
+        end
+
+        puts "\n"
```

Comment:
すごく簡潔になりましたね！列数の変更にもバッチリ対応できそうです 👍 

一応、僕だったらこう書くかもしれないかなーというのを載せておきます。参考までに。
（改行のためだけに`puts`したり、`print`を何回も実行するよりは一旦配列に詰めて`puts`一発でやった方がなんとなく良いかなー、くらいの感じなので、こちらが良いかどうかは微妙なところですが 😅 ）

```suggestion
        formatted_dir_files = array.map {|dir_file| dir_file + ' ' * (display_width - dir_file.length)}
        puts formatted_dir_files.join
```